.github/workflows/build.yml

```diff
@@ -2,7 +2,7 @@
 name: Lint and test
 on:
   pull_request:
-    branches: [main, release]
```

Comment:
Note: I do not think there is any reason not to run these tests on all PRs. Useful such as in this case when I create a PR on top of another PR.